pandas/tests/reshape/test_qcut.py

```diff
@@ -197,3 +197,30 @@ def test_date_like_qcut_bins(arg, expected_bins):
     ser = Series(arg)
     result, result_bins = qcut(ser, 2, retbins=True)
     tm.assert_index_equal(result_bins, expected_bins)
+
+
+def test_qcut_unbounded():
+    # GH 17282
+    labels = qcut(range(5), 4, bounded=False)
+    left = labels.categories.left.values
+    right = labels.categories.right.values
+    expected = np.array([-np.inf, 1.0, 2.0, 3.0, np.inf])
```

Comment:
rather than use numpy arrays, can you construct the expected Index and use ``tm.assert_index_equal``